workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -233,7 +233,7 @@ protected virtual void HandleWorkerConnectionEstablished()
         {
         }
 
-        protected virtual void HandleWorkerConnectionFailure()
+        protected virtual void HandleWorkerConnectionFailure(string errorMessage)
```

Comment:
@samiwh think it would be good to pass back the error message in case you want to display it. like we do it for the mobile clients